src/devrules/core/github_service.py

```
@@ -42,3 +47,34 @@ def fetch_pr_info(owner: str, repo: str, pr_number: int, github_config: GitHubCo
+        with yaspin(text="Updating status...", color="green"):
+            subprocess.run(
+                cmd,
+                check=True,
+                capture_output=True,
+                text=True,
+            )
+    except subprocess.CalledProcessError as e:
+        prompter.error(
+            f"Failed to update project item status: {e}",
+        )
+        raise prompter.exit(1)
```

Ruff (0.14.13)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)